src/providers/NotificationProvider/NotificationProvider.tsx

```diff
@@ -0,0 +1,41 @@
+import { TNotificationAction, TNotificationState } from './types';
+import { reducer } from './reducer';
+import { createContext, useReducer } from 'react';
```

Comment:
react imports should be on line 1

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/2/commits/ba0c3959874ad8a573436f5456e5d6253b4026d2